README.md

```diff
@@ -38,6 +38,10 @@ docker_user_home: "/home/{{ docker_user }}"
 docker_user_shell: /bin/bash
 docker_user_desired_state: present
 docker_repo_gpg_key: https://download.docker.com/linux/{{ ansible_distribution | lower }}/gpg
+docker_nonroot_users:
+  - darkwizard242
+  - ubuntu
+docker_add_nonroot_users_to_group_boolean: false
```

Comment:
This var is too long. Can u try `add_nonroot_users` ?

Reply:
I am going to set that to `docker_add_nonroot_users`. Only reason being to prepend it with relation to role in case any enduser has the same var being used elsewhere that the playbook calls. For instance, I generally have a playbook that runs multiple roles and it doesn't conflict (cause duplications) that way. Pushed [here](https://github.com/darkwizard242/ansible-role-docker/pull/8/commits/60a73d836e33e71320465d9b6222e8e02922f8f8), will be merging to master.